packages/integration-karma/test/api/getComponentDef/index.spec.js

```diff
@@ -21,6 +22,41 @@ function testInvalidComponentConstructor(name, ctor) {
     });
 }
 
+beforeEach(function() {
+    const getNormalizedFunctionAsString = fn => {
+        return fn
+            .toString()
+            .replace(/\s/g, '')
+            .replace(/\n/g, '');
+    };
+
+    jasmine.addMatchers({
```

Comment:
I don't think it's needed to add a matcher `beforeEach`. A matcher should only be added once.